service/src/main/kotlin/fi/espoo/evaka/messaging/MessageNotificationEmailService.kt

```diff
@@ -105,16 +109,16 @@ class MessageNotificationEmailService(
         return "Uusi viesti eVakassa / Nytt meddelande i eVaka / New message in eVaka$postfix"
     }
 
-    private fun getCitizenMessagesUrl(lang: Language): String {
+    private fun getCitizenMessageUrl(lang: Language, threadId: MessageThreadId?): String {
```

Comment:
`threadId` should never be null

Reply:
What do you mean? We have jobs without a saved `threadId`.

Reply:
Ah, of course 🤦 Maybe we should have versioned job definitions.